### PR TITLE
MissingIsisInterfaceAfiSafiEnable deviation remove

### PR DESCRIPTION
**Pull request by arvbaska1**
Made following changes :

- Changed Interface AFI enable to default, since similar is done at https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go#L232 for gNMI-1.2,gNMI-1.3 tests.

- Removed the deviations from metadata for RT-2.1,RT-2.2,gNMI-1.2,gNMI-1.3 for cisco

- Not removed the deviation itself since it looks to be used by other vendors for Interface/level/AFI enabled leaf here https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go#L255.

### CODE REVIEW

File: feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go [hunk 1]
```diff
@@ -86,18 +86,8 @@ func TestBasic(t *testing.T) {
 			check.Equal(isisRoot.Global().LevelCapability().State(), oc.Isis_LevelType_LEVEL_2),
 			check.Equal(port1ISIS.Enabled().State(), true),
 			check.Equal(port1ISIS.CircuitType().State(), oc.Isis_CircuitType_POINT_TO_POINT),
-		}
-
-		// if MissingIsisInterfaceAfiSafiEnable is set, ignore enable flag check for AFI, SAFI at global level
-		// and validate enable at interface level
-		if deviations.MissingIsisInterfaceAfiSafiEnable(ts.DUT) {
```
Comment (prinikasn) [inline feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go:93]: This deviation is also used by Nokia in this test, I dont think we can remove this deviation check from the test itself.  Same for otg_test

**Review by prinikasn — COMMENTED** [review]
Since CISCO does not need this deviation, can this also be removed from the following files:

 - [openconfig/featureprofiles/feature/experimental/isis/otg_tests/isis_metric_style_wide_not_enabled_test/metadata.textproto](https://github.com/openconfig/featureprofiles/blob/a6115671323d159ffe0133456b10198383202770/feature/experimental/isis/otg_tests/isis_metric_style_wide_not_enabled_test/metadata.textproto#L28)
 - [openconfig/featureprofiles/feature/experimental/isis/otg_tests/isis_interface_hello_padding_enable_test/metadata.textproto](https://github.com/openconfig/featureprofiles/blob/a6115671323d159ffe0133456b10198383202770/feature/experimental/isis/otg_tests/isis_interface_hello_padding_enable_test/metadata.textproto#L28)
 - [openconfig/featureprofiles/feature/experimental/isis/otg_tests/isis_metric_style_wide_enabled_test/metadata.textproto ](https://github.com/openconfig/featureprofiles/blob/a6115671323d159ffe0133456b10198383202770/feature/experimental/isis/otg_tests/isis_metric_style_wide_enabled_test/metadata.textproto#L28)
 - [openconfig/featureprofiles/feature/experimental/isis/ate_tests/isis_change_lsp_lifetime_test/metadata.textproto](https://github.com/openconfig/featureprofiles/blob/a6115671323d159ffe0133456b10198383202770/feature/experimental/isis/ate_tests/isis_change_lsp_lifetime_test/metadata.textproto#L28)
 - [openconfig/featureprofiles/feature/experimental/isis/otg_tests/isis_interface_passive_test/metadata.textproto](https://github.com/openconfig/featureprofiles/blob/a6115671323d159ffe0133456b10198383202770/feature/experimental/isis/otg_tests/isis_interface_passive_test/metadata.textproto#L28)

File: feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go (hunk 1 above)
Comment (arvbaska1) [reply feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go:93]: okay , i will revert this check if its failing for them.

**Comment by arvbaska1** [comment]
> Since CISCO does not need this deviation, can this also be removed from the following files:
> 
> * [openconfig/featureprofiles/feature/experimental/isis/otg_tests/isis_metric_style_wide_not_enabled_test/metadata.textproto](https://github.com/openconfig/featureprofiles/blob/a6115671323d159ffe0133456b10198383202770/feature/experimental/isis/otg_tests/isis_metric_style_wide_not_enabled_test/metadata.textproto#L28)
> * [openconfig/featureprofiles/feature/experimental/isis/otg_tests/isis_interface_hello_padding_enable_test/metadata.textproto](https://github.com/openconfig/featureprofiles/blob/a6115671323d159ffe0133456b10198383202770/feature/experimental/isis/otg_tests/isis_interface_hello_padding_enable_test/metadata.textproto#L28)
> * [openconfig/featureprofiles/feature/experimental/isis/otg_tests/isis_metric_style_wide_enabled_test/metadata.textproto ](https://github.com/openconfig/featureprofiles/blob/a6115671323d159ffe0133456b10198383202770/feature/experimental/isis/otg_tests/isis_metric_style_wide_enabled_test/metadata.textproto#L28)
> * [openconfig/featureprofiles/feature/experimental/isis/ate_tests/isis_change_lsp_lifetime_test/metadata.textproto](https://github.com/openconfig/featureprofiles/blob/a6115671323d159ffe0133456b10198383202770/feature/experimental/isis/ate_tests/isis_change_lsp_lifetime_test/metadata.textproto#L28)
> * [openconfig/featureprofiles/feature/experimental/isis/otg_tests/isis_interface_passive_test/metadata.textproto](https://github.com/openconfig/featureprofiles/blob/a6115671323d159ffe0133456b10198383202770/feature/experimental/isis/otg_tests/isis_interface_passive_test/metadata.textproto#L28)

Sure, will remove it from these test metadata files.

**Review by prinikasn — COMMENTED** [review]
/fptest virtual

**Review by nraghavendran — COMMENTED** [review]
I see that arista's deviation in being changed here. is that intentional? imo, pr should be in scope restricted for the vendor.

**Comment by arvbaska1** [comment]
> I see that arista's deviation in being changed here. is that intentional? imo, pr should be in scope restricted for the vendor.

Yes, its changed to use the existing deviation https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go#L233 for them.

**Pull request merged**